src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodsServiceSpec.scala

```diff
@@ -52,7 +50,7 @@ class MethodsServiceSpec extends FreeSpec with ScalaFutures with ScalatestRouteT
       "MethodNotAllowed error is returned" in {
         Put("/methods") ~> sealRoute(routes) ~> check {
           status should equal(MethodNotAllowed)
-          responseAs[String] === "HTTP method not allowed, supported methods: GET"
+          errorReportCheck("Agora", MethodNotAllowed)
```

Comment:
Orchestration passes all `/methods` calls to Agora regardless of full path or method.  This check indicates that the passthrough to Agora succeeded, and it's Agora returning this error, not Orchestration.  See above for the alternate scenario.